requirements/base.txt

```diff
@@ -4,6 +4,15 @@
 #
 #    make upgrade
 #
-django==1.11.27
-djangorestframework==3.9.4
+--index-url http://edx.devstack.devpi:3141/root/pypi/+simple/
```

Comment:
I’ve never seen these before. What are they from/for?

Reply:
These are an artifact of running pip-compile in devstack with its devpi server configuration.  We suppress these in edx-platform and elsewhere with some extra parameters to pip-compile: https://github.com/edx/edx-platform/blob/master/Makefile#L89